README.md

```diff
@@ -14,11 +14,11 @@
 [david-dm-dev-url]:https://david-dm.org/moxystudio/next-runtime-env?type=dev
 [david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/next-runtime-env.svg
 
-NextJS plugin to pass environment variables to NextJS's runtime configuration.
+Next.js plugin to pass environment variables to Next's runtime configuration.
```

Comment:
I believe we should use the same naming everywhere, either be it `NextJS`, `Next.js` or `Next`.

Having had a quick look through their documentation and tutorials, it's always refered as `Next.js` so in these cases:

> ... to Next's runtime configuration.

I believe we should also use `Next.js's runtime configuration.`.
Do you have reason for having both `Next.js` and `Next`  in the same sentence?

Same for the other instances.